include/host/wasi/inode.h

```diff
@@ -472,6 +473,17 @@ class INode
   /// @return Poll helper or WASI error
   static WasiExpect<Poller> pollOneoff(__wasi_size_t NSubscriptions) noexcept;
 
+#if WASMEDGE_OS_LINUX || WASMEDGE_OS_MACOS
```

Comment:
We should keep this definition, but return error about no support on windows.

Reply:
Otherwise, when we received this function call, it will report an unknown function call and then failed.

Reply:
ohhhh，thanks a lot! I will fix it.